1_fetch/out/model_RMSEs.csv

```diff
@@ -0,0 +1,116 @@
+exper_n,exper_id,model_type,rmse
```

Comment:
You can place all of the files in out directories into the .gitignore file. They do not need to be committed. You can remove them from the commit with `git rm --cached <filename>`. The cached flag ensures that the file will remain in your local repo, but removed from the git commit